source/feature-guide/feature-importing.rst

```diff
@@ -185,6 +186,19 @@ means the "metadata" object is explicitly unordered. If you wish the
 variables to have an order, you must supply an order object rather than
 relying on any order of the "metadata" object.
 
+Additionally, an optional "hidden" member is allowed, which receives a Shoji
```

Comment:
If we're changing the API to specify hidden variables this way, why would we also support indicating hiddenness as `"discarded": true` in the variable metadata? Should be one way or the other right?

Reply:
To make the transition smooth for clients, disallowing `discarded` would affect pretty much all YG's payloads.

Unless we disabled the discarded attribute when the `hidden` order is entered, that would require some extra gymnastics on our side.

Reply:
Right: if you include a "hidden" member, you're specifying hiddenness by the new API. I don't think you need to reject a payload that contains "discarded" attributes in the variables if there is a "hidden" order, but you should ignore the discarded attribute values.